### DISCUSSION TITLE
Macro Knob

### DISCUSSION THREAD

**Discussion by Qui-Gon-JooL** [SynthstromAudible/DelugeFirmware#2797] [Ideas]
Hi,

it would be great for live performances to be able to manually modify several parameters at the same time with a single knob, such as increasing the delay feedback, increasing the reverb, opening the filter, increasing the pitch of osc 2... and all that with a single knob.

Each parameter connected to the macro modulator using a patch cable may have a different value.
Which will allow each parameter to be modulated at the same time, from the value set individually with a single knob.

This is why it would be excellent to add one or more "macro modulator".
(Macro mod 1, Macro mod 2, Macro mod 3... ) juste one is already very good^^

"Macro mod" would be a modulator in the same way as an "lfo" or an "envelope" which would allow you to link several parameters to the "macro modulator" and then to be able to link this macro mod to a knob via learn, and this macro knob would allow you to modulate all the parameters that we have connected it with only one knob.

I don't know if it is physically possible to implement this ?

But that would be great, because with only two knobs in live performance it's really something I miss.

**Comment by bukru** [comment]
Sound nice, but for waiting ou can plug a midi controller.
A had an electribe 2 that i don't really use anymore, but as a midi contrôler it is really cool, 16 Channel in direct Access by pad press and 11knob per Channel, i use 1 Channel = 1 row. Really power full.

Have you try to Map the same CC to multiple parameters ? Maybe it works

**Comment by Qui-Gon-JooL** [comment]
Well in fact it's an important thing that I forgot to specify, and which is the biggest advantage of a macro mod, and which it is not possible to do with an external midi controller, thank you:

Each parameter connected to the macro modulator using a patch cable may have a different value.

which will allow each parameter to be modulated at the same time, from the value set individually with a single knob.

(I will edit my message and add this important clarification.)

**Comment by 113wsl123** [comment]
Only made an account to chip in here.
I also would highly enjoy the ability to change multiple parameters at once with one knob. The akai force has that feature and it's awesome to use that in a live set for washes etc... Macro ftw

**Comment by 113wsl123** [comment]
And it would be cool if you could jump back to the values you have programmed as the macro startpoint via pressing the knob or a shortcut..

**Comment by monkeymanford** [comment]
Yea that sounds great, especially a way to reset everything to your defaults, sometimes i get carried away and can't go back quick enough to be seamless
